docs/disaster-recovery.md

```diff
@@ -0,0 +1,442 @@
+# Failover and Failback In DR
+
+[RBD mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/)
+ is an asynchronous replication of RBD images between multiple Ceph clusters.
+This capability is available in two modes:
+
+* Journal-based: Every write to the RBD image is first recorded
+ to the associated journal before modifying the actual image.
+ The remote cluster will read from this associated journal and
+ replay the updates to its local image.
+* Snapshot-based: This mode uses periodically scheduled or
+ manually created RBD image mirror-snapshots to replicate
+ crash-consistent RBD images between clusters.
+
+Here, Snapshot based mirroring approach is used to asynchronously
```

Comment:
may be small `s` and also keep this line with `snapshot based mirroring` instead of a seperate para?

Reply:
Sure.